index.js

```diff
@@ -578,7 +569,7 @@ PersonaClient.prototype.updateProfile = function(guid, profile, token, callback)
     try {
         _.map([guid, token], function (arg) {
             if (!_.isString(arg)) {
-               throw "guid and token are required strings";
+                throw "guid and token are required strings";
```

Comment:
the `try/catch` is making this hard to read. Can it be replaced with a simple if statement?